lib/metadata.js

```diff
@@ -71,19 +70,15 @@ Metadata.prototype.updateOidcMetadata = function updateOidcMetadata(doc, next) {
 
   const self = this;
 
-  self.oidc = objectTransform({
-    source: doc,
-    map: {
-      id_token_signing_alg_values_supported: 'algorithms',
-      authorization_endpoint: 'auth_endpoint',
-    },
-    pick: [
-      'end_session_endpoint',
-      'issuer',
-      'token_endpoint',
-      'userinfo_endpoint',
-    ],
-  });
+  var oidc = {};
+  oidc.algorithms = doc.id_token_signing_alg_values_supported;
```

Comment:
I will keep the 'algorithms' one since it is a more natural name than 'id_token_signing_alg_values_supported'. In the library we used 'algorithms' everywhere and we didn't do any translation of that.